h/static/scripts/directive/test/util.js

```diff
@@ -96,8 +96,8 @@ function createDirective(document, name, attrs, initialScope, initialHtml, opts)
   opts = opts || {};
   opts.parentElement = opts.parentElement || document.body;
 
-  // create a template consisting of a single element, the directive
-  // we want to create and compile it
+  // Create a template consisting of a single element, the directive
```

Comment:
Feel free to pull obviously-safe cleanups like this into their own commits and push them directly to master to avoid noise in pull requests.